### PR TITLE
Merge maint-5.6

### PR DESCRIPTION
**Pull request by fischer-ncar**
- Add support for running Radiative Convective Equilibrium.
- Add support to archive metadata for lens experiment type.
- Make the streams file legitiment xml files so that we can parse them.
- Add support for CMIP6 OMIP (datm-pop-cice) compsets.
- Update JRA forcing files for datm.
- Update JRA025->gx1v7, JRA025->tx0.1v3 ROF2OCN mapping files.
- Update CCSM_CO2_PPMV to CMIP6 1850 value (284.317)
  in compsets with datm, POP, and POP ecosys.

Test suite:  scripts_regression_tests
Test baseline: 
Test namelist changes: 
Test status: climate changing in C & G compsets

Fixes [CIME Github issue #]

User interface changes?: 
Add a namelist for seq_flux_mct.F90 to handle the new variables.
Add an optional argument to shr_orb_cosz in shr_orb_mod.F90 to return values based on a
uniform angle.

Update gh-pages html (Y/N)?:

Code review:

### CODE REVIEW

File: src/components/data_comps/datm/cime_config/config_component.xml [hunk 1]
```diff
@@ -60,7 +60,7 @@
 
   <entry id="DATM_PRESAERO">
     <type>char</type>
-    <valid_values>none,clim_1850,clim_2000,clim_2010,trans_1850-2000,SSP1-1.9,SSP1-2.6,SSP2-4.5,SSP3-7.0,SSP4-3.4,SSP4-6.0,SSP5-3.4,SSP5-8.5,cplhist</valid_values>
+    <valid_values>none,clim_1850,clim_2000,clim_2010,trans_1850-2000,SSP1-1.9,SSP1-2.6,SSP2-4.5,SSP3-7.0,SSP4-3.4,SSP4-6.0,SSP5-3.4,SSP5-8.5,cesm2_omip,cplhist</valid_values>
```
Comment (jedwards4b) [inline src/components/data_comps/datm/cime_config/config_component.xml]: @fischer-ncar should these changes go to master?

Comment (fischer-ncar) [reply src/components/data_comps/datm/cime_config/config_component.xml]: I'll let @klindsay28 answer about including CMIP6 OMIP compsets.

Comment (jedwards4b) [reply src/components/data_comps/datm/cime_config/config_component.xml]: I think that it's okay - I realized as I looked into it further - I withdraw the question.

**Comment by klindsay28** [comment]
> I approve as long as there won't be impacts on E3SM.

@jgfouca Does E3SM have compsets using JRA forcing?
They would be impacted by aef0da4, which updates this forcing.
Also, ee892e9 and 31faeb5 would need to be extended to MPAS-O grids.

**Comment by jgfouca** [comment]
> Does E3SM have compsets using JRA forcing?

I have no idea. @amametjanov , do you know?

**Comment by amametjanov** [comment]
No, I don't. @jonbob ?

**Comment by jonbob** [comment]
Yes, we do have compsets with JRA forcing

________________________________

**Comment by jedwards4b** [comment]
@jonbob to be more precise in the question: This PR updates JRA forcing, is it acceptable to you or should we make it so that it affects cesm only?

**Comment by jonbob** [comment]
@jedwards4b - we probably would need time to look at the changes to the forcing and make any required modifications to support them.

**Comment by jedwards4b** [comment]
@jonbob - Can you review these changes now and let us know if you can support them or if we need to bring them in in a different way so that the old JRA forcing is still available?

**Comment by jonbob** [comment]
@jedwards4b - I asked around and we do need to introduce the new JRA forcing data in a way that does not disrupt our current JRA-based compsets...

**Comment by jedwards4b** [comment]
@jonbob thank you for clarifying.  We will pull those changes from the PR and resubmit with a new name so that previous JRA forcing capability is maintained.

**Comment by jonbob** [comment]
@jedwards4b - thanks and sounds good

**Comment by fischer-ncar** [comment]
Closing this PR, will open another one with the JRA forcing changes removed.

**Pull request closed without merging**